tests/core/test_email.py

```diff
@@ -1,29 +1,133 @@
+        call_command("notifymail", "--cron", "--no-sys-exit")
+
+        assert len(mail.outbox) == 1
+        assert (
+            mail.outbox[0].subject
+            == "You have new notifications from example.com (type: instantly)"
+        )
+        assert (
+            mail.outbox[0].body
+            == """Dear alice,
+
+These are notifications sent instantly from example.com.
+
+ * Test is a test
+
+
+Thanks for using our site!
+
+Sincerely,
+example.com
+
+"""
+        )
+
+    def test_notify_with_url(self):
+        mail.outbox = []
+
+        # Subscribe User 1 to test key
+        utils.subscribe(self.user1_settings, self.TEST_KEY, send_emails=True)
+        utils.notify("Test is a test", self.TEST_KEY, url="/test")
+
+        call_command("notifymail", "--cron", "--no-sys-exit")
+
+        assert len(mail.outbox) == 1
+        assert (
+            mail.outbox[0].subject
+            == "You have new notifications from example.com (type: instantly)"
+        )
+        assert (
+            mail.outbox[0].body
+            == """Dear alice,
+
+These are notifications sent instantly from example.com.
+
+ * Test is a test
+   https://example.com/test
 
-    def test_notify_two_users(self):
 
-        # Subscribe User 2 to test key
-        utils.subscribe(self.user2_settings, self.TEST_KEY)
-        utils.subscribe(self.user1_settings, self.TEST_KEY)
-        utils.notify("Another test", self.TEST_KEY)
+Thanks for using our site!
+
+Sincerely,
+example.com
+
+"""
+        )
+
+    @override_settings(
+        NYT_EMAIL_TEMPLATE_NAMES=OrderedDict(
+            {NotifyTestBase.TEST_KEY: "testapp/notifications/email.txt"}
+        ),
+        NYT_EMAIL_SUBJECT_TEMPLATE_NAMES=OrderedDict(
+            {NotifyTestBase.TEST_KEY: "testapp/notifications/email_subject.txt"}
+        ),
+    )
+    def test_custom_email_template(self):
+        # TODO: https://github.com/django-wiki/django-nyt/issues/124
+        from django_nyt import settings as nyt_settings
+
+        nyt_settings.EMAIL_TEMPLATE_NAMES = OrderedDict(
+            {NotifyTestBase.TEST_KEY: "testapp/notifications/email.txt"}
+        )
+        nyt_settings.EMAIL_SUBJECT_TEMPLATE_NAMES = OrderedDict(
+            {NotifyTestBase.TEST_KEY: "testapp/notifications/email_subject.txt"}
+        )
+
+        mail.outbox = []
+
+        # Subscribe User 1 to test key
+        utils.subscribe(self.user1_settings, self.TEST_KEY, send_emails=True)
+        utils.notify("Test is a test", self.TEST_KEY, url="/test")
+
+        call_command("notifymail", "--cron", "--no-sys-exit")
+
+        assert len(mail.outbox) == 1
+        assert mail.outbox[0].subject == "subject"
+        assert mail.outbox[0].body == "Test\n"
+
+        # Reset to default state
+        # TODO: https://github.com/django-wiki/django-nyt/issues/124
+        nyt_settings.EMAIL_TEMPLATE_NAMES = OrderedDict()
+        nyt_settings.EMAIL_SUBJECT_TEMPLATE_NAMES = OrderedDict()
```

Comment:
This is rapidly becoming unbearable but I don't want to overload this PR.

I suggest that we get it fixed in the same release, though.